Automattic/wp-calypso
# Add a new section on themes empty results to upgrade the plan

#### Proposed Changes

* Add a section to upsell the Business plan when no result is found
* Add a button that redirects to the cart with the business plan
* Add a track event for the upgrade plan button

<img width="1288" alt="Screen Shot 2022-10-06 at 18 08 08" src="https://user-images.githubusercontent.com/5039531/194434524-36e71055-a4b7-4a0e-8bd3-e1d915e9a02c.png">
 


#### Testing Instructions
* Go to the themes page on a site with a free plan  (`/themes/{free_site}`)
* Search for a term that shouldn't return any theme. Ex: `not found`
* Check if the section with an upgrade button is shown.

---

* Go to the themes page on a site with a paid plan(`/themes/{paid_sote}`)
* Search for a term that shouldn't return any theme. Ex: `not found`
* Check if the section with an upgrade button is similar to what is currently in production


#### Pre-merge Checklist






Fixes #68285
